### ISSUE TITLE
Deploy the latest fixes to testnet.mdw.aepps.com

### ISSUE THREAD

**Issue by davidyuk** [aeternity/aepp-middleware#174]
For me, requests from #154 still have different values (1114 vs 1171). Probably the corresponding fix is not deployed yet.

**Comment by johnsnewby** [comment]
Yeah there have been no deployments for a bit as I went away and we wanted to keep things stable while I was not there to keep an eye. Expect a big release late this week, including a big set of naming fixes

**Comment by davidyuk** [comment]
https://testnet.aeternal.io/middleware/transactions/account/ak_2swhLkgBPeeADxVTAVCJnZLY5NZtCFiM93JxsEaMuC59euuFRQ/count outputs:
```
{"count":1476}
```
but
```js
(await
  (await
    fetch(
      'https://testnet.aeternal.io/middleware/transactions/account/ak_2swhLkgBPeeADxVTAVCJnZLY5NZtCFiM93JxsEaMuC59euuFRQ'
    )
  ).json()
).length
```
evaluates to `1536`
